tests/integration/test_script_runner.py

```diff
@@ -364,6 +364,12 @@
         user_flow_script="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sqlite_data.py",
         data_context_dir="tests/integration/fixtures/no_datasources/great_expectations",
     ),
+    IntegrationTestFixture(
```

Comment:
Recently we split up test definitions by backend so we could parallelize / isolate in `tests/integration/test_definitions/`, could you move this to `tests/integration/test_definitions/postgresql/integration_tests.py`?

Reply:
@TrangPham  @anthonyburdi and I agreed that I will refactor this once my PRs merge -- and then everybody can use it.  If you want to merge yours before the refactor, please let me know (may be fewer merge conflicts for you to resolve this way).  So if you can address the questions regarding this PR soon, I am happy to re-review right away.  Thanks!